DockerFiles/AzureDevOpsAgents/jmeter-agent.Dockerfile

```diff
@@ -0,0 +1,64 @@
+#Developed from https://github.com/justb4/docker-jmeter/blob/master/Dockerfile (commit 6aa034c6c362f8e29cb81f7d51637560d29b2f24)
+FROM ubuntu:18.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN echo "APT::Get::Assume-Yes \"true\";" > /etc/apt/apt.conf.d/90assumeyes
+
+ARG JMETER_VERSION="5.1.1"
+ENV JMETER_HOME /opt/apache-jmeter-${JMETER_VERSION}
+ENV	JMETER_BIN	${JMETER_HOME}/bin
+ENV	JMETER_DOWNLOAD_URL  https://archive.apache.org/dist/jmeter/binaries/apache-jmeter-${JMETER_VERSION}.tgz
+
+# Change TimeZone TODO: TZ still is not set!
```

Comment:
This comment - ARG is for when the container is being build, ENV is for future runs.
Could this be a case of just using the wrong command here?

Reply:
Changed as suggested